addons/revolutionaries/CfgVehicles_Russian.hpp

```diff
@@ -0,0 +1,161 @@
+// Units - Revolutionaries (Russian)
+
+class TACU_Revolutionaries_U_O_Russian_Base: TACU_Main_U_OPFOR_Soldier_Base {
+    author = "Jack";
+    displayName = "Rifleman (FAL)";
+    faction = "TACU_Revolutionaries_O";
+    scope = 2;
+    scopeCurator = 2;
+    identityTypes[] = {"CUP_D_Language_RU", "Head_Euro", "G_Balaclava_blk"};
+    genericNames = "CUP_Names_RussianMen";
+    icon = "iconMan";
+    role = "Rifleman";
+    uniformClass = "CUP_U_C_Citizen_01";
+    backpack = "TACU_Revolutionaries_Backpack_FNFAL";
+    linkedItems[] = {"ItemWatch","ItemCompass","ItemMap"};
+    respawnLinkedItems[] = {"ItemWatch","ItemCompass","ItemMap"};
+    Items[] = {mag_5("ACE_fieldDressing"),"ACE_EarPlugs"};
+    respawnItems[] = {mag_5("ACE_fieldDressing"),"ACE_EarPlugs"};
+    weapons[] = {"CUP_arifle_FNFAL"};
+    respawnWeapons[] = {"CUP_arifle_FNFAL"};
+    magazines[] = {mag_2("CUP_20Rnd_762x51_FNFAL_M")};
+    respawnMagazines[] = {mag_2("CUP_20Rnd_762x51_FNFAL_M")};
+    editorSubcategory = "TACU_Revolutionaries_EdSubCat_O_Russian";
+    // editorPreview = QPATHTOF(ui\Cartel_U_O_Soldado_Rifleman.jpg);
+    headgearList[] = {};
+    TACU_allowedFacewear[] = {
+        "G_Balaclava_blk", 1
+    };
+};
+
+//Unit Specifics
+
+class TACU_Revolutionaries_U_O_Russian_Rifleman01: TACU_Revolutionaries_U_O_Russian_Base {
+    displayName = "Rifleman (MK14)"; 
+    scope = 2;
+    scopeCurator = 2;
+    uniformClass = "CUP_U_C_Citizen_02";
+    weapons[] = {"srifle_DMR_06_hunter_F"};
+    respawnWeapons[] = {"srifle_DMR_06_hunter_F"};
+    magazines[] = {mag_2("10Rnd_Mk14_762x51_Mag")};
+    respawnMagazines[] = {mag_2("10Rnd_Mk14_762x51_Mag")};
+	backpack = "TACU_Revolutionaries_Backpack_Mk14";
```

Comment:
Most "backpack" entries have tabs and are formatted incorrectly.
```suggestion
    backpack = "TACU_Revolutionaries_Backpack_Mk14";
```